### ISSUE TITLE
Password For "admin" For Access Control

### ISSUE THREAD

**Issue by jlpoolen** [kellerben/dudle#6]
When creating a poll, you have an option to control access by clicking the "Access Control" tab. I tried entering "test" for the two password fields in the screen shot below and an authentication window popped open which would not accept "test" as the password for user "admin".

The README has no mention of "admin" -- it would be helpful to have some documentation on this area.  Is there a default password that is user configurable?  I grepped the tree of ruby files using "admin" and "password" and did not find any hints on the subject.
![2015-11-19 12_05_55-dudle - test20 - access control](https://cloud.githubusercontent.com/assets/227313/11283078/b4053854-8eb7-11e5-846c-f03eedfdd85d.png)

Where is the password configurable?

**Comment by kellerben** [comment]
The password for user admin should be test afterwards…
Could you please paste the contents of Test20/.ht*

**Comment by jlpoolen** [comment]
themis dudle # cat Test20/.ht*
<Files ~ "^(edit_columns|invite_participants|access_control|delete_poll).cgi$">
AuthType digest
AuthName "dudle-Test20"
AuthUserFile "/usr/local/src/dudle/Test20/.htdigest"
Require user admin
ErrorDocument 401 /dudle/authorization_required.cgi?user=admin&poll=Test20
</Files>
admin:dudle-Test20:d6d9b110a5084ecedfb9cd011cf3957b
themis dudle #

**Comment by jlpoolen** [comment]
themis dudle # cat Test20/.ht*
    <Files ~ "^(edit_columns|invite_participants|access_control|delete_poll).cgi$">
    AuthType digest
    AuthName "dudle-Test20"
    AuthUserFile "/usr/local/src/dudle/Test20/.htdigest"
    Require user admin
    ErrorDocument 401 /dudle/authorization_required.cgi?user=admin&poll=Test20
    </Files>
    admin:dudle-Test20:d6d9b110a5084ecedfb9cd011cf3957b
    themis dudle #

**Comment by jlpoolen** [comment]
I revisited the page, same browser same session of FireFox, and tried admin/test again, but the password challenge dialog keeps popping up and I finally have to click Cancel to unlock my browser.

**Comment by jlpoolen** [comment]
Just for the record;

     themis dudle # ls -la Test20/.ht*
     -rw-r--r-- 1 apache apache 276 Nov 19 12:03 Test20/.htaccess
     -rw-r--r-- 1 apache apache  52 Nov 19 12:03 Test20/.htdigest
     themis dudle #

**Comment by jlpoolen** [comment]
I tried using another browser, Chrome, and am tailing the apache error log.  When I clicked the Access Control Tab, this appeared in the error log:


     [Thu Nov 19 19:54:11 2015] [error] [client 192.168.1.2] access to /dudle/Test20/access_control.cgi 
     failed, reason: verification of user id '<null>' not configured, referer: http://themis/dudle/Test20/

This occurred before the Authentication window appeared.

![2015-11-19 19_56_27-dudle - test20 - poll](https://cloud.githubusercontent.com/assets/227313/11291698/2a5922be-8ef8-11e5-8e54-67403429e936.png)

When I enter "admin/test", the same error message appears when I click Log In.

**Comment by kellerben** [comment]
I think this is your problem:
https://github.com/kellerben/dudle/blob/99921fc3db66427fd5200cae4043cd369e1cc4b5/README#L27-L29
(you need to enable the apache mod digest-auth)

**Comment by jlpoolen** [comment]
You are correct.  I forgot to add into my http.conf file:

    LoadModule auth_digest_module modules/mod_auth_digest.so

I had read that instruction, determined that my apache instance did not have the auth_digest module compiled, so I recompiled apache to include it.  But, I must have been distracted and forgot to do the other part which was to add the LoadModule line.

Note: with the new Apache instance running, I was unable to go back to previous polls that were created without the benefit of the module running and access them.  When I created a new poll, then the authentication facility worked.  So, the auth_digest needs to be running at the time of of the poll's creation, activating auth_digest does not necessarily fix previously created polls.  I do not need access to the previous polls, they are all tests; I'm just mentioning this in case someone in the future made the same mistake I did.

Thank you very much.  I think everything is now working.  My focus on this project is piecemeal since I'm doing this outside of my employment.
